src/Math-Tests-Random/PMPoissonGeneratorTest.class.st

```diff
@@ -4,6 +4,14 @@ Class {
 	#category : #'Math-Tests-Random'
 }
 
+{ #category : #tests }
+PMPoissonGeneratorTest >> generateASampleOfSize: sampleSize UsingGenerator: gen [
```

Comment:
I'd go for `#generateASampleOfSize:usingGenerator:` (with a lower-case 'u'), which is more in line with what I see elsewhere in Pharo.